superset-frontend/src/SqlLab/actions/sqlLab.js

```diff
@@ -663,6 +664,36 @@ export function setActiveSouthPaneTab(tabId) {
   return { type: SET_ACTIVE_SOUTHPANE_TAB, tabId };
 }
 
+export function toggleLeftBar(queryEditor) {
+  const hideLeftBar = !queryEditor.hideLeftBar;
+  return function (dispatch) {
+    const sync = isFeatureEnabled(FeatureFlag.SQLLAB_BACKEND_PERSISTENCE)
+      ? SupersetClient.put({
```

Comment:
can we use `makeApi` here instead of `SupersetClient` directly

Reply:
@hughhhh I'd rather leave this consistent with the rest of the code that manages `SQLLAB_BACKEND_PERSISTENCE`, if that's ok, until we have time to refactor the whole thing. Then we could maybe create a custom hook to manage the whole state.